eng/pipelines/tasks/CSharp.yml

```diff
@@ -0,0 +1,24 @@
+steps:
+- task: NuGetToolInstaller@0
```

Comment:
I would remove the `NuGetToolInstaller` and `NuGetCommand` tasks since we already install the nuget tool and run restore on the CI Build.

See https://github.com/microsoft/MIEngine/blob/e65e5a71eeaefaa5f116836a27fd4bc227f6bf2e/eng/pipelines/templates/Build.template.yml#L10

Reply:
There is no need to run NuGetToolInstall@0 since it is already defined earlier in the build.